.github/workflows/restart.yml

```diff
@@ -1,94 +1,24 @@
 ---
 name: restart application
 
-on:
+on:  # yamllint disable-line rule:truthy
   workflow_dispatch:
   schedule:
-    - cron: '7/15 * * * *'
+    - cron: '7/15 * * * *'  # every 15 mins
 
 jobs:
   restart-staging:
-    concurrency: staging
     name: restart (staging)
-    environment: staging
-    runs-on: ubuntu-latest
-    env:
-      APP_URL: https://inventory-stage-datagov.app.cloud.gov
-    steps:
-      - name: checkout
-        uses: actions/checkout@v3
-      - name: restart
-        uses: cloud-gov/cg-cli-tools@main
-        with:
-          command: cf restart inventory --strategy rolling
-          cf_org: gsa-datagov
-          cf_space: staging
-          cf_username: ${{secrets.CF_SERVICE_USER}}
-          cf_password: ${{secrets.CF_SERVICE_AUTH}}
-      - name: smoke test
-        run: bin/smoke.sh
-      - name: restart proxy
-        uses: cloud-gov/cg-cli-tools@main
-        with:
-          command: cf restart inventory-proxy --strategy rolling
-          cf_org: gsa-datagov
-          cf_space: staging
-          cf_username: ${{secrets.CF_SERVICE_USER}}
-          cf_password: ${{secrets.CF_SERVICE_AUTH}}
-      - name: Create Issue if it fails 😢
-        if: ${{ failure() }}
-        uses: JasonEtco/create-an-issue@v2
-        env:
-          GITHUB_TOKEN: ${{ secrets.ADD_TO_PROJECT_PAT }}
-          GITHUB_JOB: ${{ toJson(github)['job'] }}
-          GITHUB_ATTEMPTS: ${{ github.run_attempt }}
-          LAST_COMMIT: ${{ github.sha }}
-          LAST_RUN_BY: ${{ github.actor }}
-          RUN_ID: ${{ github.run_id }}
-        with:
-          filename: .github/restart_failure.md
-          assignees: ${{ github.actor }}
-          update_existing: true
+    uses: gsa/data.gov/.github/workflows/app-restart-template.yml@main
+    with:
+      environ: staging
+      app_names: "{\"include\":[{\"app\":\"inventory\"},{\"app\":\"inventory-proxy\"},]}"
```

Comment:
does this need to be an array?

Reply:
this run looks like it surfaces the error
https://github.com/GSA/inventory-app/actions/runs/4000944795

Reply:
I wish. 

It's the horrible solution that was found for [catalog deploy](https://github.com/GSA/catalog.data.gov/blob/c0bee863a3594f1f1418c006d097739bde4ee14a/.github/workflows/publish.yml#L81) that I am just copying (note: the solution is clever but it reads horribly). I tried some yaml shenanigans, but it is a weird yaml in json in yaml thing that apparently needs to be in this format.

If you can find a better, more readable option please put it in. I hate things like this.

Reply:
> this run looks like it surfaces the error https://github.com/GSA/inventory-app/actions/runs/4000944795

That run was from [this commit](https://github.com/GSA/inventory-app/pull/536/commits/aeb28ad2cf23091995cd18d30b939e0d2f4374d7). Note the commit message.

Reply:
ok. i gotcha. will the JSON break if there's a trailing comma? i see one here, but not in catalog-deploy.

Reply:
looks like it doesn't. voila!

Reply:
It looks like it accepts it, but good catch cause there was a `prod` vs `production` issue.

Reply:
For evidence: [successful run](https://github.com/GSA/inventory-app/actions/runs/4001082009)